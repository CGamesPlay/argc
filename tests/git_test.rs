use insta::assert_snapshot;

#[test]
fn test_git() {
    assert_argc!(include_str!("git.sh"), &["git", "-h"]);
}

#[test]
fn test_git_add() {
    assert_argc!(include_str!("git.sh"), &["git", "add", "-h"]);
}

#[test]
fn test_git_remote() {
    assert_argc!(include_str!("git.sh"), &["git", "push", "-h"]);
}

#[test]
fn test_git_log() {
    assert_argc!(include_str!("git.sh"), &["git", "log", "-h"]);
}
