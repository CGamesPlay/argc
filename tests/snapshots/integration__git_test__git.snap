---
source: tests/git_test.rs
assertion_line: 6
expression: output

---
RUN
git -h

STDOUT
exit 1

STDERR
git 2.17.1
nobody <nobody@example.com>
A fictional versioning CLI

USAGE:
    git [OPTIONS] [SUBCOMMAND]

OPTIONS:
        --git-dir <GIT-DIR>    
    -h, --help                 Print help information
        --no-pager             
    -p, --paginate             
    -V, --version              Print version information

SUBCOMMANDS:
    add     Add file contents to the index
    help    Print this message or the help of the given subcommand(s)
    log     Shows the commit log.
    push    Update remote refs along with associated objects


